pennylane/math/multi_dispatch.py

```diff
@@ -370,6 +370,11 @@ def dot(tensor1, tensor2, like=None):
 
         return np.tensordot(x, y, axes=[[-1], [-2]], like=like)
 
+    import scipy  # pylint: disable=import-outside-toplevel
+
+    if scipy.sparse.issparse(x):
+        return x.dot(y)

```

Comment:
```suggestion
    if like="scipy":
        return x.dot(y)

```
sparse matrices should be the only type of data with `like="scipy"`.

Reply:
When I was trying to add a test for multidispatching dot, I realized that multiple places need to be adjusted for this purpose, e.g. `get_interface`, order of branch-out scipy tensor etc. Since rn we don't even need `dot` for pipeline, I'll remove the dot dispatch for now and again add a story to backlog todo

Reply:
[sc-85329](https://app.shortcut.com/xanaduai/story/85329)
[sc-85330](https://app.shortcut.com/xanaduai/story/85330)